src/main/java/it/gov/pagopa/payhub/activities/activity/classifications/ClearClassifyIufActivityImpl.java

```diff
@@ -0,0 +1,32 @@
+package it.gov.pagopa.payhub.activities.activity.classifications;
+
+import it.gov.pagopa.payhub.activities.dao.ClassifyDao;
+import it.gov.pagopa.payhub.activities.utility.Utilities;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Component;
+
+/**
+ * Implementation for defining an activity to delete classifications based on IUF.
+ */
```

Comment:
it's not required to have javadoc on implementation, they will inherit it from the interface

Reply:
OK, deleted